ouroboros-consensus-diffusion/src/ouroboros-consensus-diffusion/Ouroboros/Consensus/Network/NodeToNode.hs

```diff
@@ -612,10 +611,10 @@ mkApps kernel Tracers {..} mkCodecs ByteLimits {..} genChainSyncTimeout lopBucke
       bracketWithPrivateRegistry
         (chainSyncHeaderServerFollower
            (getChainDB kernel)
-           ( case isPipeliningEnabled version of
-              ReceivingTentativeBlocks    -> ChainDB.TentativeChain
-              NotReceivingTentativeBlocks -> ChainDB.SelectedChain
-           )
+           ChainDB.TentativeChain
+           -- ^ isPipeliningenabled was removed from o-n-api,
+           -- can we remove this argument, or eliminate the type
+           -- itself unless it is used for other purposes, eg. tests?
```

Comment:
We want to keep the `ChainType` type (of which `TentativeChain` is a constructor) as it is used for different kinds of `Follower`s, but the `ChainType` argument can be removed from `chainSyncHeaderServerFollower`.

Reply:
Resolved as in https://github.com/IntersectMBO/ouroboros-consensus/pull/1262#discussion_r1798970918